frontend/components/cards/ModuleUnionCard.tsx

```diff
@@ -8,6 +9,27 @@ interface ModuleUnionCardProps {
   selected?: boolean;
 }
 
+interface HighlightEllipsisProps {
+  unsafeHTML: string;
+}
+
+const HighlightEllipsis: FC<HighlightEllipsisProps> = ({ unsafeHTML }) => (
+  <Box
+    sx={{
+      "& *": { fontSize: "0.8rem", backgroundColor: "rgba(0, 0, 0, 0)" },
```

Comment:
@aaw0151, can you also update this line to render all elements inline?  Like so:

`"& *": { fontSize: "0.8rem", backgroundColor: "rgba(0, 0, 0, 0)", display: "inline" },`